lib/board.rb

```diff
@@ -0,0 +1,92 @@
+class Board
+  attr_reader :cells, :height
+  def initialize(height = 4, width = 4)
+    @height = height
+    @width = width
+    @cells = generate_cells
+  end
+
+  def board_letters
+    ("A"..("A".ord + @height - 1).chr).to_a
```

Comment:
This was confusing for me so...

("A"..("A".ord + @height - 1).chr).to_a
("A"..(65 + @height - 1).chr).to_a
("A"..(65 + 4 - 1).chr).to_a
("A"..(68).chr).to_a
("A".."D").to_a
["A","B","C","D"]

I didn't know we could do this with ord and chr. Interesting.